pyfr/backends/openmp/kernels/axnpby.mako

```diff
@@ -11,31 +11,27 @@ axnpby(int nrow, int nblocks,
     #pragma omp parallel for
     for (int ib = 0; ib < nblocks; ib++)
     {
-        int idx, blksz = nrow*BLK_SZ*${ncola};
         #pragma omp simd
-        for (int i = 0; i < blksz; i++)
-        {
-            idx = i + ib*blksz;
-            x0[idx] = ${pyfr.dot('a{l}', 'x{l}[idx]', l=nv)};
-        }
+        for (int i = ib*nrow*BLK_SZ*${ncola}; i < (ib + 1)*nrow*BLK_SZ*${ncola}; i++)
```

Comment:
given that `ib` is always multiplied by nrow in this loop, could you change the in for loop stride?

Reply:
I think that would work, although the variable would be something of an oddball as it would not represent the block number or the total number of elements in a block.